sales_team_invoiced_target_domain/model/crm_team.py

```diff
@@ -0,0 +1,30 @@
+import ast
+
+from odoo import fields, models
+from odoo.osv import expression
+
+
+class CrmTeam(models.Model):
+    _inherit = "crm.team"
+
+    def _compute_invoiced(self):
+        if not self:
+            return
+        sales_team_invoiced_domain = self.env.company.sales_team_invoiced_domain
+        if not sales_team_invoiced_domain:
+            return super()._compute_invoiced()
+        today = fields.Date.today()
+        invoiced_domain = [
+            ("move_type", "in", ["out_invoice", "out_refund", "out_receipt"]),
+            ("team_id", "in", self.ids),
+            ("date", ">=", fields.Date.to_string(today.replace(day=1))),
+            ("date", "<=", fields.Date.to_string(today)),
```

Comment:
Converting to string is unnecessary; it's better to use native date types.

```suggestion
            ("date", ">=", today.replace(day=1)),
            ("date", "<=", today),
```